css-flexbox/reset.css

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+#card-deck {
+  display: flex;
+  flex-wrap: wrap;
+}
```

Comment:
And this.

Reply:
hmm. i just went and tried to delete the 2 but they were already removed

Reply:
alright 
both fixed